app/controllers/subject_viewer.coffee

```diff
@@ -69,13 +81,17 @@ class SubjectViewer extends Controller
     if @classification
       @classification.bind 'change', @onClassificationChange
       @classification.bind 'add-species', @onClassificationAddSpecies
-
-      @html template @classification
-
-      @active = Math.floor @classification.subject.location.standard.length / 2
-      @activate @active
-
-      @onClassificationChange()
+      deferred = Experiments.getCohortRetriever()
```

Comment:
Promises or deferreds can resolve to values. I'd like to see this turn into something like

```coffee
Experiments.getCohort()
  .then (cohort) =>
    if cohort?
      @classification.metadata.cohort = cohort
  .always =>
    # Do the stuff in "setClassificationFinalPart"
```